clarifies percentile in agg field for mesures

mirnawong1 opened the pull request:
Updates the description for the agg in the top table by adding percentile. Also adds add'l info on support sql engines for agg_params

Resolves  #5398


added table based on [pr here](https://github.com/dbt-labs/metricflow/pull/395) provided by @tlento

Commit:
add percentile

Commit:
add table

Commit:
clarify

Commit:
add add'l context and remove metric_time from example

mirnawong1 committed:
Update website/docs/docs/build/dimensions.md

mirnawong1 committed:
Update website/docs/docs/build/dimensions.md

Commit:
update filter

mirnawong1 committed:
Merge branch 'current' into add-percentile

mirnawong1 committed:
Merge branch 'add-percentile' into update-dimensions-scd

tlento commented on website/docs/docs/build/measures.md:
```diff
@@ -56,8 +56,32 @@ Supported aggregations include:
+agg: percentile
+agg_params:
+  percentile: .99
+  use_discrete_percentile: False  # False calculates the discrete percentile, True calculates the continuous percentile
```
This comment is backwards - False calculates continuous, True calculates discrete.

mirnawong1 replied:
thanks so much for flagging this @tlento ! have updated the examples 🙏

tlento commented on website/docs/docs/build/measures.md:
```diff
@@ -137,7 +161,7 @@ semantic_models:
         agg_params:
           percentile: .99
-          use_discrete_percentile: False #False will calculate the discrete percentile and True will calculate the continuous percentile
+          use_discrete_percentile: False # False calculates the discrete percentile and True calculates the continuous percentile
```
Oh I see, I think this comment was backwards originally.

tlento reviewed (commented):
Overall looks great, I noted a backwards comment from the original documentation inline. Thanks!

nghi-ly commented on website/docs/docs/build/measures.md:
```diff
@@ -56,8 +56,32 @@ Supported aggregations include:
-| percentile        | Percentile calculation across the values  |
+| percentile        | Percentile calculation across the values. |
 
+**Percentile aggregation example**<br />
```
sugg: promote to level H4 heading  since it already reads and looks like a heading and also makes the example linkable/shareable

```suggestion
#### Percentile aggregation example
```

nghi-ly commented on website/docs/docs/build/measures.md:
```diff
@@ -56,8 +56,32 @@ Supported aggregations include:
+| percentile        | Percentile calculation across the values. |
 
+**Percentile aggregation example**<br />
+If you're using the `percentile` aggregation, you must use the `agg_params` field under it to specify details for the percentile aggregation (such as what percentile to calculate and whether to use discrete or continuous calculations).
```
sugg for conciseness 

```suggestion
If you're using the `percentile` aggregation, you must use the `agg_params` field to specify details for the percentile aggregation (such as what percentile to calculate and whether to use discrete or continuous calculations).
```

nghi-ly commented on website/docs/docs/build/measures.md:
````diff
@@ -56,8 +56,32 @@ Supported aggregations include:
+  use_discrete_percentile: False  # False calculates the discrete percentile, True calculates the continuous percentile
+```
+
+**Percentile across supported engine types**<br />
````
sugg: promote to level H4 heading

```suggestion
#### Percentile across supported engine types
```

nghi-ly commented on website/docs/docs/build/measures.md:
````diff
@@ -56,8 +56,32 @@ Supported aggregations include:
+```
+
+**Percentile across supported engine types**<br />
+The following table displays which SQL engine supports continuous, discrete, approximate, continuous, and approximate discrete percentiles.
````
```suggestion
The following table lists which SQL engine supports continuous, discrete, approximate, continuous, and approximate discrete percentiles.
```

nghi-ly reviewed (commented):
looks great! left minor suggs

mirnawong1 committed:
Update website/docs/docs/build/dimensions.md

Co-authored-by: Thomas Lento <tlento@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/dimensions.md

Co-authored-by: Thomas Lento <tlento@users.noreply.github.com>

mirnawong1 committed:
update dimensions (#5409)

this pr updates the dimensions doc:
- add more info on valid_from and valid_to 
- adds implementation guidelines for scd ii tables
- updates the dimension name in the first code example.

help provided by @tlento [in slack
convo](https://dbt-labs.slack.com/archives/C050Z6UP4RK/p1714413611285839?thread_ts=1714402934.686409&cid=C050Z6UP4RK)



Resolves #5395
Resolves #5396

mirnawong1 committed:
Update website/docs/docs/build/measures.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/measures.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/measures.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update website/docs/docs/build/measures.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into add-percentile

mirnawong1 committed:
Update dimensions.md

mirnawong1 committed:
Update measures.md

mirnawong1 commented:
thanks so much @nghi-ly - can you rereview pls?

mirnawong1 committed:
Merge branch 'current' into add-percentile
